_layouts/download.html

```diff
@@ -75,7 +75,17 @@ <h2>Install Scala with <span style="font-weight: bold;">cs setup</span> (recomme
             </a>
           </div>
         </div>
-        <h2>Other ways to install Scala</h2>
+        
+	<div id="version-choice">
+          <h2>Which version of Scala should I choose?</h2>
+          <p>There are 2 distribution lines of Scala 3:</p>
+          <ul>
+            <li><strong>Scala Next</strong> - The <strong>default</strong> to be used by most users, containing the latest features, bug fixes and improvements.</li>
```

Comment:
this perhaps can display the current LTS/Next version?

Reply:
That's a good idea, I'm not sure how to implement it with this templating engine though :disappointed:

Reply:
its jekyll, maybe you can look here (not necessary imo):

https://github.com/scala/scala-lang/blob/37767b2ab89aa360f2442b29e3f6eab501965ae3/_layouts/frontpage.html#L48-L62

Reply:
Thank you Jamie, that helped! the updated layout renders version numbers as on this screenshot:

![obraz](https://github.com/user-attachments/assets/6e3e21a2-fac2-4feb-a66b-06752742b75f)

@lrytz it also removes the sethian parentheses :wink:

Reply:
nice!